app/views/register/trustees/TrusteesAnswerPageView.scala.html

```diff
@@ -22,15 +22,15 @@
     formHelper: FormWithCSRF
 )
 
-@(index : Int, draftId: String, answerSections: Seq[Section])(implicit request: Request[_], messages: Messages)
+@(index : Int, draftId: String, answerSections: Seq[Section], messageKeyPrefix: String)(implicit request: Request[_], messages: Messages)
 
 @main_template(
     title = messages("trusteesAnswerPage.title")
     ) {
 
     @components.back_link()
 
-    @components.heading("trusteesAnswerPage.heading")
```

Comment:
@JoshuaH01 can you uses string interpolation here instead?
`@components.heading(s"$messageKeyPrefix.heading")`